Makefile

```diff
@@ -126,13 +126,29 @@ build:
 		.
 
 ################################################################################
-# Scan                                                                         #
+# Image security                                                               #
 ################################################################################
 
 .PHONY: scan
 scan:
 	grype $(DOCKER_IMAGE_NAME):$(IMMUTABLE_DOCKER_TAG) -f medium
 
+.PHONY: generate-sbom
+generate-sbom:
+	syft $(DOCKER_IMAGE_NAME):$(IMMUTABLE_DOCKER_TAG) \
+		-o spdx-json \
+		--file ./artifacts/dockerhub-gateway-$(VERSION)-SBOM.json
```

Comment:
nit: can we rename these `brigade-dockerhub-gateway-...`?  It seems the precedence is to prefix the gateway names with `brigade`.

Reply:
Yes! That is what I intended and you caught the mistake. Fixing.